### PR TITLE
Add comment-based Checkstyle suppression

### PR DESCRIPTION
**Pull request by nmiyake**
Adds SuppressWithNearbyCommentFilter to checkstyle.xml with
a custom check style. Addresses issue #51.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Gradle Baseline Java Changelog
 
+## 0.2.3
+- add ability to suppress Checkstyle rules using comments
```
Comment (uschi2000) [inline CHANGELOG.md]: can you actually this file? using github release notes now.

Comment (nmiyake) [inline CHANGELOG.md]: Didn't catch that -- think that formatting erased the verb :). What should I do instead (delete file, not edit file, something else)?

Comment (uschi2000) [inline CHANGELOG.md]: "delete"

Comment (uschi2000) [inline CHANGELOG.md]: I may not have typed that word ...

**Comment by nmiyake** [comment]
@uschi2000 changed to built-in Checkstyle approach based on our conversation earlier today

**Pull request merged**